package.json

```diff
@@ -32,12 +32,12 @@
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "env-cmd -f .env react-scripts start",
-    "build": "env-cmd -f .env.production react-scripts build",
+    "start": "env-cmd -f .env.development.local react-scripts start",
+    "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "start:local": "env-cmd -f .env.local react-scripts start",
-    "start:production": "env-cmd -f .env.production react-scripts start"
+    "start:local": "env-cmd -f .env.development.local react-scripts start",
+    "start:production": "env-cmd -f .env.production.local react-scripts start"
```

Comment:
since the env files are removed, we need to have the files shared to each of us to run locally. Maybe you can share the files in discord, or have some guidance for everyone to do. I tried this branch locally, and I'm unable to run the app cause the env files are deleted 😬